hack/module.sh

```diff
@@ -48,7 +48,7 @@ function main() {
   'unittest' | 'unittests' | 'units' | 'unit')
     KUBE_CACHE_MUTATION_DETECTOR=${kube_cache_mutation_detector} \
     KUBE_PANIC_WATCH_DECODE_ERROR=${kube_panic_watch_decode_error} \
-    go test -short -race ./...
+    go test -short ./...
```

Comment:
Don't merge this to `main`!

Reply:
See https://github.com/vmware-tanzu/pinniped/issues/2157 for some context on this. I'm curious to see how the integration tests fare with K8s 1.32.

Reply:
This has been removed